app/views/layouts/_header.html.erb

```diff
@@ -0,0 +1,13 @@
+	<header class="cd-header">
+		<div id="cd-logo"><a href="/home/index">Sam Rebelsky's Curriculum Planner</a></div>
+
+		<nav class="cd-main-nav">
+			<ul>
+				<!-- inser more links here -->
+				<li><a href='/users/sign_in'>Students</a></li>
```

Comment:
Since we are just using the dashboard, can we combine these two links (`Students` and `Sam`)? The combined link should point to the dashboard.